sdk/devcenter/azure-developer-devcenter/samples/create_environment_sample.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# The MIT License (MIT)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the ""Software""), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+# --------------------------------------------------------------------------
+import logging
+import os
+
+from azure.developer.devcenter import DevCenterClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+def main():
+    logging.basicConfig(level=logging.DEBUG)
+    LOG = logging.getLogger()
+
+    # Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+    # AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, DEVCENTER_NAME
+    try:
+        tenantId = os.environ["AZURE_TENANT_ID"]
+    except KeyError:
+        LOG.error("Missing environment variable 'AZURE_TENANT_ID' - please set if before running the example")
+        exit()
+
+    try:
+        devCenterName = os.environ["DEVCENTER_NAME"]
```

Comment:
same with https://github.com/Azure/azure-sdk-for-python/pull/26696/files#r998894421